bridge/contracts/Federation.sol

```diff
@@ -164,6 +230,42 @@ contract Federation is Initializable, UpgradableOwnable {
         return true;
     }
 
+  function acceptTransfer(
+    address originalTokenAddress,
+    address payable sender,
+    address payable receiver,
+    uint256 number,
+    bytes32 blockHash,
+    bytes32 transactionHash,
+    uint32 logIndex,
+    TokenType tokenType
+  ) internal {
+    if (tokenType == TokenType.NFT) {
+      require(bridgeNFT != INFTBridge(0), "Federation: Empty bridgeNFT");
```

Comment:
Idk how solidity works on this, but can't we use the COIN token type directly instead of 0?

Reply:
maybe just `TokenType.COIN` like in this example? (then we could also remove the `@dev` comment clearing up that COIN == 0 and NFT == 1)
![image](https://user-images.githubusercontent.com/7085857/130103536-87015b1a-4bca-46e0-9d75-6bb63a529091.png)

Reply:
Actually he is comparing that the bridgeNFT address is set, if its 0 it means that is not  and throw and error